Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=16.20.2-bookworm-slim
+ARG NODE_VERSION=22.4.0-bookworm-slim
```

Comment:
there is already 22.14.0-bullseye-slim, we should use it.

or should we switch to the newer 22.14.0-bookworm-slim ?

Reply:
```suggestion
ARG NODE_VERSION=22.14.0-bookworm-slim
```